doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -171,23 +171,87 @@ is typically done in the REST interface)
+  is described in the JSON Attribute Representation section below. Obviously, `id` and `type` are not allowed as attribute names.
+
+An example of this syntax in shown below:
+
+
+```
+{
+  "id": "entityID",
+  "type": "entityType",
+  "attr_1": <val_1>,
+  "attr_2": <val_2>,
+  ...
+  "attr_N": <val_N>
+}
+```
+
+
+## JSON Attribute Representation
+
+An attribute is represented by a JSON object with the following syntax:
+
+* The attribute value is specified by the `value` property, which value can be any JSON datatype.
+
+* The attribute NGSI type is specified by the `type` property, which value will be a string containing the attribute NGSI type.
+
+* The attribute metadata is specified by the `metadata` property. Its value is another JSON object which will contain a
+  property per metadata element defined (the name of the property is the `name` of the metadata element). Each metadata element,
+  in turn, is represented by a JSON object containing the following properties:
+
+  * `value`: Its value will contain the metadata value, which can correspond to any JSON datatype.
+
+  * `type`: Its value will contain a string representation of the metadata type name.
+
+An example of this syntax in shown below:
+
+```
+{
+  "value": <...>,
+  "type": <...>,
+  "metadata": <...>
+}
+```
+
+## Simplified Entity Representation
+
+There are two representation modes, which must be supported by implementations, and which allow to generate simplified representations of entities.
+
+* keyValues mode. This mode represents each entity attribute only by its value, leaving out the information about type and metadata.
+  Thus, when this mode is on, only the attribute value is represented and will correspond to the value of the property which represents the attribute.
+  See example below (To be provided).
+
+* values mode. This mode represents the entity only by a collection of attribute values conveyed by an Array. Id and type information are left out.
+  See example below (To be provided).
+
+## Partial representations
+
+Some operations imply that only a partial representation of an entity will be provided, which are summarized below:
+
+* `id` and `type` are forbidden in update operations, as they are inmutable properties.
+
+* When not forbidden, `type` can be omitted in requests. Abstent `type` means that the type is `null`.
+
+* Attribute/metadata `value` can be omitted in requests, meaning that the attribute/metadata has `null` value. It is always present in responses.
+
+* Attribute/metadata `type` can be ommited in requests, meaning that the attribute/metadata doesn't have a NGSI type. In responses, this
+  property is set to `null` if the attribute doesn't have a type.
+
+* Attribute `metadata` can be ommited in requests, meaning that the attribute doesn't have metadata. In responses, this property is set to
+  `{}` if the attribute doesn't have metadata.
+
```

Comment:
might be omitted by request payloads. In that particular case it will mean that there are no metadata properties associated to the attribute. Responses must include this property, which value will be an empty JSON object (``{}``) if no metadata is present.